src/elements/netspeak-graph.tsx

```diff
@@ -0,0 +1,778 @@
+import React, { useEffect, useState } from "react";
+import { Phrase, Netspeak, ReadonlyNetspeakSearchResult, WordTypes } from "../lib/netspeak";
+import { NetspeakGraphBody } from "./netspeak-graph-body";
+import Popup from "reactjs-popup";
+import gearIcon from "../img/gearicon.svg";
+import "./netspeak-graph.scss";
+import Switch from "react-switch";
+import Slider from "rc-slider";
+import "rc-slider/assets/index.css";
+import Select from "react-select";
+import { PhraseState } from "./netspeak-result-list";
+import { read } from "fs";
```

Comment:
Trying to read local files isn't a good idea on a website.

Reply:
Do you mean "import { read } from "fs";"? I think ESlint added that one automatically.

Or another line? AFAIK, these sites should be a part of the build. :x

Reply:
Yes, the [`fs` import](https://nodejs.org/api/fs.html#fs_file_system).

Also, ESLint does add imports (at least, I'm not aware of any rules that do). I think you might have had a variable called "read-something" and your IDE auto-imported `fs`. Happens sometimes.

Just remove the import, it's unused anyways.

Reply:
Alright, thanks.
That's what I meant, the IDE auto-added it because of ESLint.